pom.xml

```diff
@@ -5,7 +5,19 @@
   <groupId>scientific-java</groupId>
   <artifactId>ScientificJavaLib</artifactId>
   <version>0.0.1-SNAPSHOT</version>
-  <packaging>jar</packaging>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>8</source>
```

Comment:
I think you want source and target to be 1.8

Reply:
Good spots. Should be sorted now.